control-plane-operator/controllers/hostedcontrolplane/scheduler/config.go

```diff
@@ -43,7 +44,10 @@ func generateConfig() (string, error) {
 			Kubeconfig: kubeConfigPath,
 		},
 		LeaderElection: componentbasev1.LeaderElectionConfiguration{
-			LeaderElect: pointer.BoolPtr(true),
+			LeaderElect:   pointer.BoolPtr(true),
```

Comment:
Are we passing or needing to pass these values to any other controller?
In any case to avoid accidental future skew and ease reviews it'd be good to have per package consts:
```
// https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#handling-kube-apiserver-disruption
recommendedLeaseDuration = "137s"
recommendedRenewDeadline = "107s"
recommendedRetryPeriod = "26s"
```

Then you can `time.ParseDuration` where you need to use them.